CHANGELOG.md

```diff
@@ -1,15 +1,32 @@
 # Changelog
 
-## 0.11.0 - Unreleased
+## 0.12.0 - Unreleased
```

Comment:
This line should be added after release.

Reply:
Updated.